### PR TITLE
Fix Bug 1454263 - fix search drop down dividers and hover colors

### PR DESCRIPTION
**Pull request by rlr**
r? @Mardak 

Before:
(hovering the change search settings button)
screen shot 2018-04-16 at 12 49 56 pm

(hovering ebay search engine)
screen shot 2018-04-16 at 12 49 51 pm
screen shot 2018-04-16 at 12 49 30 pm

After:
(hovering the change search settings button)
screen shot 2018-04-16 at 12 45 37 pm

(hovering ebay search engine)
screen shot 2018-04-16 at 12 45 50 pm
screen shot 2018-04-16 at 12 46 10 pm

### CODE REVIEW

File: system-addon/content-src/components/Search/_Search.scss [hunk 1]
```diff
@@ -122,6 +123,21 @@
       }
     }
 
+    .contentSearchOneOffItem {
+      background-image: none;
```
Comment (rlr) [inline system-addon/content-src/components/Search/_Search.scss]: Not sure why images were being used as dividers, that code must be super old. Hacking over it here with plans to fix it in m-c when we update `contentSearchUI` for other coming changes (change the settings button to a gear icon like the awesome bar). It does make the hover background color not fill the space vertically but good enough for now says Amy.

File: system-addon/content-src/components/Search/_Search.scss [hunk 2]
```diff
@@ -78,6 +78,7 @@
   // Adjust the style of the contentSearchUI-generated table
   // sass-lint:disable-block class-name-format
   .contentSearchSuggestionTable {
+    background-color: var(--newtab-search-dropdown-color);
     border: 0;
     box-shadow: $context-menu-shadow;
     transform: translateY(3px);
```
Comment (Mardak) [inline system-addon/content-src/components/Search/_Search.scss]: let's push this down 1px more now that the outline is 4px, so `translateY(4px)`

**Comment by Mardak** [comment]
Probably add to _variables

**Comment by as-pine-proxy** [comment]
[Treeherder: 328487bd56cff758b39206fde603ae5e4e5f90af](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=328487bd56cff758b39206fde603ae5e4e5f90af)

**Pull request merged**